lang/RStudio/RStudio.def

```diff
@@ -0,0 +1,45 @@
+Bootstrap: library
+From: debian:9
+
+%environment
+    LC_ALL="C"
+    export LC_ALL
+
+%post
+    apt-get update
+    apt-get -y install systemd systemd-sysv gdebi-core procps libssl1.1 ed wget curl libqt5webkit5 libqt5core5a
+    apt-get -y install r-base r-base-dev
```

Comment:
These 2 lines (10 and 11) can be merged into one for `apt-get -y install`

Reply:
This was done to separate dependencies, from the application itself.